.github/workflows/main.yml

```diff
@@ -8,7 +8,6 @@ on:
   push:
     branches: [ master-isik-stufe-2 ]
```

Comment:
1. hier ggf. noch ändern zu [main-isik-stufe-3] und auch für Stufe 2 und 1?
2. ... gleichzeitig frage ich mich: wir haben doch direct PUSH auf main geblockt (hoffentlich)? Kann dann Das "push: ..." nicht einfach weg?
3. Wir sollten das gleich für alle repos umsetzen

Reply:
1.: wie in Teams besproche aktiviere ich den build für alle branches
2.: man sieht das protected zeichen neben den main branches in der branch Übersicht. Da ich keine Adminrechte habe sehe ich aber das zugrundeliegende RuleSet hier nicht. Dies können nur Projektadmins (ebenso können nur diese branches protecten)
3.: kann ich gerne übernehmen